precompiles/ArbOwner.go

```diff
@@ -38,72 +38,72 @@ func (con ArbOwner) RemoveChainOwner(c ctx, evm mech, addr addr) error {
 	return c.State.ChainOwners().IsMember(addr)
 }
 
-// Retrieves the list of chain owners
+// GetAllChainOwners retrieves the list of chain owners
 func (con ArbOwner) GetAllChainOwners(c ctx, evm mech) ([]common.Address, error) {
 	return c.State.ChainOwners().AllMembers(65536)
 }
 
-// Set how slowly ArbOS updates its estimate of the L1 basefee
+// SetL1BaseFeeEstimateInertia sets how slowly ArbOS updates its estimate of the L1 basefee
 func (con ArbOwner) SetL1BaseFeeEstimateInertia(c ctx, evm mech, inertia uint64) error {
 	return c.State.L1PricingState().SetInertia(inertia)
 }
 
-// Sets the L2 gas price directly, bypassing the pool calculus
+// SetL2BaseFee sets the L2 gas price directly, bypassing the pool calculus
 func (con ArbOwner) SetL2BaseFee(c ctx, evm mech, priceInWei huge) error {
 	return c.State.L2PricingState().SetBaseFeeWei(priceInWei)
 }
 
-// Sets the minimum base fee needed for a transaction to succeed
+// SetMinimumL2BaseFee sets the minimum base fee needed for a transaction to succeed
 func (con ArbOwner) SetMinimumL2BaseFee(c ctx, evm mech, priceInWei huge) error {
 	return c.State.L2PricingState().SetMinBaseFeeWei(priceInWei)
 }
 
-// Sets the computational speed limit for the chain
+// SetSpeedLimit sets the computational speed limit for the chain
 func (con ArbOwner) SetSpeedLimit(c ctx, evm mech, limit uint64) error {
 	return c.State.L2PricingState().SetSpeedLimitPerSecond(limit)
 }
 
-// Sets the maximum size a tx (and block) can be
+// SetMaxTxGasLimit sets the maximum size a tx (and block) can be
 func (con ArbOwner) SetMaxTxGasLimit(c ctx, evm mech, limit uint64) error {
 	return c.State.L2PricingState().SetMaxPerBlockGasLimit(limit)
 }
 
-// Set the L2 gas pricing inertia
+// SetL2GasPricingInertia sets the L2 gas pricing inertia
 func (con ArbOwner) SetL2GasPricingInertia(c ctx, evm mech, sec uint64) error {
 	return c.State.L2PricingState().SetPricingInertia(sec)
 }
 
-// Set the L2 gas backlog tolerance
+// SetL2GasBacklogTolerance sets the L2 gas backlog tolerance
 func (con ArbOwner) SetL2GasBacklogTolerance(c ctx, evm mech, sec uint64) error {
 	return c.State.L2PricingState().SetBacklogTolerance(sec)
 }
 
-// Gets the network fee collector
+// GetNetworkFeeAccount gets the network fee collector
 func (con ArbOwner) GetNetworkFeeAccount(c ctx, evm mech) (addr, error) {
 	return c.State.NetworkFeeAccount()
 }
 
-// Gets the infrastructure fee collector
+// GetInfraFeeAccount gets the infrastructure fee collector
 func (con ArbOwner) GetInfraFeeAccount(c ctx, evm mech) (addr, error) {
 	return c.State.InfraFeeAccount()
 }
 
-// Sets the network fee collector
+// SetNetworkFeeAccount sets the network fee collector
 func (con ArbOwner) SetNetworkFeeAccount(c ctx, evm mech, newNetworkFeeAccount addr) error {
 	return c.State.SetNetworkFeeAccount(newNetworkFeeAccount)
 }
 
-// Sets the network fee collector
+// SetInfraFeeAccount sets the network fee collector
```

Comment:
Commentary like this doesn't provide any additional context or information. 

I don't have a helpful suggestion for you, but anything in the format of `// DoesSomething does something.` is redundant